### DISCUSSION TITLE
Overview of offline commands

### DISCUSSION THREAD

**Discussion by RoemIko** [CrowdStrike/falconpy#749] [Q&A]
Hi there,

Is there a place where i can find which commands work on offline hosts?
Because at the moment I have found out that `get` and `put-and-run` don't work on hosts that are offline queued.

Another side note: 
https://www.falconpy.io/Service-Collections/Real-Time-Response-Admin.html#available-base-commands 
`put-and-run` is not in the list of available commands but it does work on online hosts 😊

**Comment by jshcodes** [comment]
Hi @RoemIko -

I'm researching the first part of this question and will reply back here once I have more detail.

Thank you for letting us know about the `put-and-run` command!  We'll get the [documentation updated](https://github.com/CrowdStrike/falconpy/issues/751) to reflect this. 😄

**Comment by jshcodes** [comment]
Hi @RoemIko -

Can we see a bit of your code that isn't working?  Our testing shows `put-and-run` as a viable command to execute against a queued_offline host. (We're working on an example for `get` now.)
